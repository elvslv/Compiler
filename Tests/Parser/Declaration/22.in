const t = 12.3456;
var a: record a: integer; end;
function m(a: record t: integer; end;): integer; forward;