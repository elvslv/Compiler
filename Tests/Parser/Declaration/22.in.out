const T = 12.3456;
var A : record 
    var A : INTEGER;
end;

34 3 Invalid type